core/src/main/scala/org/apache/spark/deploy/JsonProtocol.scala

```diff
@@ -210,22 +210,59 @@ private[deploy] object JsonProtocol {
    *         `status` status of the master,
    *         see [[org.apache.spark.deploy.master.RecoveryState.MasterState]]
    */
-  def writeMasterState(obj: MasterStateResponse): JObject = {
+  def writeMasterState(field: Option[String], obj: MasterStateResponse): JObject = {
```

Comment:
Shall we update the annotation, too?

Reply:
Do you mean the documentation?

Reply:
I mean the `@return` part above

Reply:
Sure~

Reply:
Uh, I missed the signature. Could we put `field: Option[String]` as the last parameter?
It seems give a default `None` is better.

Reply:
Sure. Thanks.